### ISSUE TITLE
npm is not updated

### ISSUE THREAD

**Issue by mentaman** [PolCPP/nativescript-downloadmanager#3]
Hey, I see there is a difference between the github version which has 
`downloadFile(url: string, options:DownloadOptions, cb: Function);`
with lots of options in options. and the npm version has 
`downloadFile(url: string, cb: Function, directory?: string, filename?: string, title?: string, description?: string): void;`

I installed it using: 
`tns plugin add nativescript-downloadmanager`

**Comment by PolCPP** [comment]
Yeah, i probably forgot to update after a merging a pull request. Will do it when i get back from the states, thanks for the reminder!

**Comment by bnbs** [comment]
@PolCPP can you please update the npm version?

**Comment by bhavincb** [comment]
is it updated?
